test/metabase/util/schema_test.clj

```diff
@@ -119,3 +121,154 @@
             schema {(s/optional-key :thing) s/Int
                     validator               s/Any}]
         (is (nil? (s/check (su/open-schema schema) (assoc value :random-thing :whatever))))))))
+
+(defn- plumatic-validate
+  [schema x]
+  (boolean (u/ignore-exceptions
+             (s/validate schema x))))
+
+(defn- malli-validate
+  [schema x]
+  (boolean (u/ignore-exceptions
+             (mc/validate schema x))))
```

Comment:
```clojure
(mc/validate int? 1)
;; => true
(mc/validate int? "x")
;;=> false
```

`mc/validate` does the same thing as `malli-validate`, so you could use mc/validate directly or write malli-validate as:

```suggestion
(defn- malli-validate [schema x] (mc/validate schema x))
```

Reply:
I did this at first, but the `u.date/parse` call in TemporalStringMalli was throwing.